benchmark/attri_util.py

```diff
@@ -180,14 +182,23 @@ def __str__(self) -> str:
                 f"{'Size Detail':>20}\n"
                 f"{'-' * 120}\n"
             )
+        elif self.result[0].gbps is not None:
+            header = (
```

Comment:
`tflops` and  `gpbs` can be set simultaneously

Reply:
I agree.

Reply:
Can we choose metric in a per test-case way?

Reply:
> Can we choose metric in a per test-case way?

Yes, by setting the `--metrics gbps` parameter when using `pytest`